src/onedrive.d

```diff
@@ -629,8 +630,22 @@ final class OneDriveApi
 		try {
 			http.perform();
 		} catch (CurlException e) {
-			// Potentially Timeout was reached on handle error
-			log.error("ERROR: There was a timeout in accessing the Microsoft OneDrive service - Internet connectivity issue?");
+			// Parse and display error message received from OneDrive
+			log.error("ERROR: OneDrive returned an error with the following message:");
+			auto errorArray = splitLines(e.msg);
+			string errorMessage = errorArray[0];
+			
+			if (canFind(errorMessage, "Couldn't connect to server on handle")) {
+				// This is a curl timeout
+				log.error("  Error Message: There was a timeout in accessing the Microsoft OneDrive service - Internet connectivity issue?");
+				// Must exit here
+				log.error("Exiting application due to Internet connectivity issues");
+                exit(-1);
```

Comment:
Spacing seems to be strange here

Reply:
Thats weird as I have checked the file locally & the spacing is 100% fine ... odd ... replacing with 'spaces'

Reply:
If you look at the file online the spacing is right .. Microsoft / Github must have changed something in the API - as the spacing is right ??

https://github.com/abraunegg/onedrive/blob/Issue-%23693/src/onedrive.d